openlibrary/macros/SubjectTags.html

```diff
@@ -1,10 +1,10 @@
-$def with(work,location,tags=[])
+$def with(work, tags=[])
 
 $def render_subjects(label, subjects, track_value, prefix=""):
   $if subjects:
     <div class="section link-box">
-      <h6>$label</h6>
-      <span>
+      <span class="clamp">
+	<h6>$label</h6>
```

Comment:
```suggestion
      <span class="clamp">
      <h6>$label</h6>
```
Replace tab with spaces.